Assets/Mirror/Runtime/NetworkReaderPool.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Mirror
+{
+    public static class NetworkReaderPool
+    {
+        static readonly Stack<NetworkReader> pool = new Stack<NetworkReader>();
+
+        public static NetworkReader GetReader(byte[] bytes)
+        {
+            UnityEngine.Debug.Log($"NetworkReaderPool:GetReader byte[] {pool.Count}");
```

Comment:
using UnityEngine would make this shorter in two places

Reply:
this is fine for debugging,  but we obviously want to remove this before merging

Reply:
Yes I intend those 3 debug log lines to be removed